mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping

```diff
@@ -2,8 +2,19 @@ CLASS net/minecraft/class_2794 net/minecraft/world/gen/chunk/ChunkGenerator
 	COMMENT In charge of shaping, adding biome specific surface blocks, and carving chunks,
 	COMMENT as well as populating the generated chunks with {@linkplain net.minecraft.world.gen.feature.Feature features} and {@linkplain net.minecraft.entity.Entity entities}.
 	COMMENT Biome placement starts here, however all vanilla and most modded chunk generators delegate this to a {@linkplain net.minecraft.world.biome.source.BiomeSource biome source}.
-	FIELD field_12761 biomeSource Lnet/minecraft/class_1966;
+	FIELD field_12761 effectiveSource Lnet/minecraft/class_1966;
+		COMMENT effectiveSource is used for feature/structure generation, while actualSource is what is saved and displayed.
```

Comment:
I see the rationale behind the names. There might be a better name to consider for these.

Reply:
Hmm, as an idea: could rename `actualSource` to just `source`, and this field to something along the lines of `populationSource` or `populationOverwriteSource`?